corehq/apps/settings/views.py

```diff
@@ -385,6 +386,19 @@ def dispatch(self, request, *args, **kwargs):
         return super(TwoFactorPhoneSetupView, self).dispatch(request, *args, **kwargs)
 
 
+class TwoFactorPhoneDeleteView(BaseMyAccountView, PhoneDeleteView):
+    success_url = "/account/two_factor/"
+
+    def get_success_url(self):
+        messages.add_message(self.request, messages.SUCCESS, "Phone number added!")
+        return resolve_url(self.success_url)
```

Comment:
I mentioned this in the other PR too, but I think it would be better to reference `TwoFactorProfileView.urlname` and use `reverse` for this.

Reply:
Done: https://github.com/dimagi/commcare-hq/pull/19577/commits/baba8ada3f38a21e9d737e0af0d1fac32cba33da
(Also changed the message to say "Phone number removed")